### PR TITLE
cnv-4078: release note for unpause in UI

### PR DESCRIPTION
**Pull request by ousleyp**
Jira: https://issues.redhat.com/browse/CNV-4078
Preview build: https://cnv-4078--ocpdocs.netlify.com/openshift-enterprise/latest/cnv/cnv_release_notes/cnv-release-notes.html#web-console-improvements

### CODE REVIEW

**Comment by jelkosz** [comment]
LGTM

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -15,6 +15,12 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == New and changed features
 
+=== Web console improvements
+
+* You can now view the *Paused* status of a virtual machine on the web console. The web console displays this status on the *Virtual Machines* dashboard and on the *Virtual Machine Details* page.
```
Comment (aburdenthehand) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:20]: `s/on/in/` the web console.

**Comment by aburdenthehand** [comment]
One nit, otherwise LGTM

**Comment by kbidarkar** [comment]
LGTM

**Comment by kbidarkar** [comment]
Just wonder, should we mention in release notes explicitly that, currently PAUSED state of a VM can only be viewed from the web-console ? 
Otherwise things look good.

**Comment by ousleyp** [comment]
> Just wonder, should we mention in release notes explicitly that, currently PAUSED state of a VM can only be viewed from the web-console and not from the CLI side ?
> Otherwise things look good.

@kbidarkar Thanks for the review! There is no way at all to tell that a VM is paused from the CLI? Yes, I think we might want to mention that somewhere, especially because you can *only* do the pause action from the CLI.
cc @slintes -- thoughts?

**Comment by kbidarkar** [comment]
I think we should, because currently, we are making the user shift between CLI and UI, to pause and unpause.
Use CLI: To pause `Web-UI has no option to pause currently`
Use Web-UI: To Unpause `To know which VM's to unpause, first. As we can't get this info from CLI`

**Comment by slintes** [comment]
> I think we should, because currently, we are making the user shift between CLI and UI, to pause and unpause.
> Use CLI: To pause `Web-UI has no option to pause currently`

Which I still don't understand.... :man_shrugging: 

> Use Web-UI: To Unpause `To know which VM's to unpause, first. As we can't get this info from CLI`

You can: paused VMs have a condition of type `Paused` in their status, see an example of querying it from CLI here: https://kubevirt.io/user-guide/#/usage/life-cycle?id=pausing-and-unpausing-a-virtual-machine

**Comment by kbidarkar** [comment]
> > I think we should, because currently, we are making the user shift between CLI and UI, to pause and unpause.
> > Use CLI: To pause `Web-UI has no option to pause currently`
> 
> Which I still don't understand.... man_shrugging
> 
> > Use Web-UI: To Unpause `To know which VM's to unpause, first. As we can't get this info from CLI`
> 
> You can: paused VMs have a condition of type `Paused` in their status, see an example of querying it from CLI here: https://kubevirt.io/user-guide/#/usage/life-cycle?id=pausing-and-unpausing-a-virtual-machine

Cool! I didn't know.

Then should we mention this query example in release notes? `get vmi testvm -o=jsonpath='{.status.conditions[?(@.type=="Paused")].message}'`

**Comment by ousleyp** [comment]
> > > I think we should, because currently, we are making the user shift between CLI and UI, to pause and unpause.
> > > Use CLI: To pause `Web-UI has no option to pause currently`
> > 
> > 
> > Which I still don't understand.... man_shrugging
> > > Use Web-UI: To Unpause `To know which VM's to unpause, first. As we can't get this info from CLI`
> > 
> > 
> > You can: paused VMs have a condition of type `Paused` in their status, see an example of querying it from CLI here: https://kubevirt.io/user-guide/#/usage/life-cycle?id=pausing-and-unpausing-a-virtual-machine
> 
> Cool! I didn't know.
> 
> Then should we mention this query example in release notes? `get vmi testvm -o=jsonpath='{.status.conditions[?(@.type=="Paused")].message}'`

@kbidarkar @slintes Yes, I agree that we should put this command in the release notes and/or docs. I'm thinking about where might be the best place to put it... and I also think it would be a good idea to file a usability bug about the fact that you can only unpause from the UI (unless one already exists). I'll try to get to this early next week.

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 2]
```diff
@@ -26,6 +26,21 @@ include::modules/technology-preview.adoc[leveloffset=+2]
...
+
+* You can now view the *Paused* status of a virtual machine in the web console. The web console displays this status on the *Virtual Machines* dashboard and on the *Virtual Machine Details* page. +
+
+[NOTE]
+====
+You can also view the *Paused* status of a virtual machine by using the `oc` CLI:
+----
+$ oc get vmi testvm -o=jsonpath='{.status.conditions[?(@.type=="Paused")].message}'
+----
+====
```
Comment (ousleyp) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:39]: @kbidarkar do you think this is ok? It doesn't really belong under "web console improvements" but I want to keep the relevant info together. cc @aburdenthehand if you want to weigh in :) thanks!

**Comment by ousleyp** [comment]
@slintes @kbidarkar I have created the usability bug, if you would like to follow it: https://bugzilla.redhat.com/show_bug.cgi?id=1826474

**Comment by ousleyp** [comment]
> @slintes @kbidarkar I have created the usability bug, if you would like to follow it: https://bugzilla.redhat.com/show_bug.cgi?id=1826474

Ah, it was closed. (The lack of pause functionality was intentional.)

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 2 above)
Comment (kbidarkar) [reply cnv/cnv_release_notes/cnv-release-notes.adoc:39]: @ousleyp: Yes looks good now.

**Comment by kbidarkar** [comment]
LGTM

**Pull request merged**